### PR TITLE
Fix: Avoid NPE when MDC contains null values

### PR DESCRIPTION
**Pull request by wreulicke**
## :scroll: Description

Avoid NPE when MDC contains null values
Follow up https://github.com/getsentry/sentry-java/pull/1364

## :pencil: Checklist

- [x] I reviewed the submitted code
- [x] I added tests to verify the changes
- [x] I updated the docs if needed
- [x] No breaking changes

### CODE REVIEW

**Comment by marandaneto** [comment]
@wreulicke why is it a follow-up of #1364 ? doesn't #1364 fix the issue at all?

**Comment by wreulicke** [comment]
@marandaneto 
To be exact, This fixes the issues similar to #1364.
`follow up` might not be accurate.

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 * Feat: Add option to ignore exceptions by type (#1352)
 * Fix: Fix NPE when MDC contains null values (#1364)
+* Fix: Avoid NPE when MDC contains null values(#1385)
```
Comment (marandaneto) [inline CHANGELOG.md:5]:
```suggestion
* Fix: Fix NPE when MDC contains null values (sentry-logback) (#1364)
* Fix: Avoid NPE when MDC contains null values (sentry-jul) (#1385)
```

**Comment by marandaneto** [comment]
> @marandaneto
> To be exact, This fixes the issues similar to #1364.
> `follow up` might not be accurate.

I just noticed its pretty much the same fix but for sentry-jul, that was the issue, I thought you were just fixing the same issue for sentry-logback :)

mind writing a unit test? thanks

**Comment by wreulicke** [comment]
@marandaneto

I have written the unit test.
Could you check this PR?

**Review by marandaneto — APPROVED** [review]
LGTM

**Comment by marandaneto** [comment]
> @marandaneto
> 
> I have written the unit test.
> Could you check this PR?

all good, thanks for the contribution @wreulicke

**Pull request merged**